Makefile

```diff
@@ -7,6 +7,8 @@ VERSION_REPLACES ?= 0.93.0
 
 DEPLOY_DIR ?= manifests
 
+export DEPLOY_CERT_MANAGER ?= true
```

Comment:
why do we need this ? didn't we say we don't want it ?

Reply:
it is deployed by the cluster-sync helper scripts, not by CNAO core itself
https://github.com/kubevirt/cluster-network-addons-operator/pull/1808/files#diff-557d9a4f5525f40b226019817998304cf7e07b6394d0d0508ce18847aea2f6c3R27

since we always have ipam as part of the default CR we need it as part of cluster-sync
if someone deploys a custom CR he can just disable `export DEPLOY_CERT_MANAGER=false`